src/api/WazzupServerLib.js

```diff
@@ -4,6 +4,56 @@
  *
  */
 
-function getChats(chatId, userId, offset, count) {}
+/**
+ * Function for user to login
+ *
+ * @param { string } EmailID
+ * @param { string } Password
+ *
+ */
+//TODO: Test fetchWithTimeout functionality with backend
```

Comment:
Remove Todo

Reply:
fetch with timeout was not yet tested, only general fetch was tested with backend. Added withTimeOut newly in case server is down/not responding.